src/SmartComponents/RunTaskModal/ParameterTextbox.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import {
+  Form,
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextInput,
+} from '@patternfly/react-core';
+
+const ParameterTextbox = ({
+  parameter,
+  updateParameters,
+  customDescription = undefined,
+}) => {
+  const [parameterText, setParameterText] = useState(
+    parameter.default || parameter.value
+  );
```

Comment:
Same question here if we could remove this state variable completely and stick to the `parameter.value` for the current parameter value.

Reply:
Same as my explanation above